packages/admin/cms-admin/src/dependencies/DependencyList.tsx

```diff
@@ -0,0 +1,245 @@
+    forceRefresh?: boolean;
+};
+
+interface DependencyListProps {
+    query: TypedDocumentNode<Query, QueryVariables>;
+    variables: Record<string, unknown>;
+}
+
+const pageSize = 10;
+
+export const DependencyList = ({ query, variables }: DependencyListProps) => {
+    const intl = useIntl();
+    const entityDependencyMap = useDependenciesConfig();
+    const contentScope = useContentScope();
+    const apolloClient = useApolloClient();
+    const history = useHistory();
+
+    const dataGridProps = useDataGridRemote({ queryParamsPrefix: "dependencies", pageSize: pageSize });
+
+    const { data, loading, error, refetch } = useQuery<Query, QueryVariables>(query, {
+        variables: {
+            offset: dataGridProps.page * dataGridProps.pageSize,
+            limit: dataGridProps.pageSize,
+            ...variables,
+        },
+    });
+
+    const columns: GridColDef<DependencyItem>[] = [
+        {
+            field: "nameInfo",
+            headerName: intl.formatMessage({ id: "comet.dependencies.dataGrid.nameAndInfo", defaultMessage: "Name/Info" }),
+            sortable: false,
+            flex: 1,
+            renderCell: ({ row }) => {
+                return (
+                    <sc.NameInfoWrapper>
+                        <sc.NameInfoTypography color="text.primary">{row.name}</sc.NameInfoTypography>
+                        <sc.NameInfoTypography color="text.secondary">{row.secondaryInformation}</sc.NameInfoTypography>
+                    </sc.NameInfoWrapper>
+                );
+            },
+        },
+        {
+            field: "type",
+            headerName: intl.formatMessage({ id: "comet.dependencies.dataGrid.type", defaultMessage: "Type" }),
+            sortable: false,
+            renderCell: ({ row }) => <sc.StyledChip label={entityDependencyMap[row.graphqlObjectType]?.displayName ?? row.graphqlObjectType} />,
+        },
+        {
+            field: "actions",
+            headerName: "",
+            sortable: false,
+            renderCell: ({ row }) => {
+                const dependencyObject = entityDependencyMap[row.graphqlObjectType] as DependencyInterface | undefined;
+
+                if (dependencyObject === undefined) {
+                    console.warn(
+                        `Cannot log URL because no implementation of DependencyInterface for ${row.graphqlObjectType} was provided via the DependenciesConfig`,
+                    );
+                    return <FormattedMessage id="comet.dependencies.dataGrid.cannotLoadUrl" defaultMessage="Cannot determine URL" />;
+                }
+
+                const loadUrl = () => {
+                    return dependencyObject.getUrl({
+                        rootColumnName: row.rootColumnName,
+                        jsonPath: row.jsonPath,
+                        contentScopeUrl: contentScope.match.url,
+                        apolloClient,
+                        id: row.id,
+                    });
+                };
```

Comment:
Loading the URL on demand is a UX problem because we have to use buttons instead of links. That means opening many dependencies in new tabs after another (CMD + Click) isn't possible. 

But I think we'll live with that for now

Reply:
Could probably be solved by a sub-route created by DependencyList that calls getUrl and redirects. But as you say - not that important for now.

Reply:
We could load the Url on hover, but this is definitely out of scope 😁